jsx/src/components/ServerDashboard/ServerDashboard.jsx

```diff
@@ -258,7 +258,7 @@ const ServerDashboard = (props) => {
           )}
         </td>
         <td data-testid="user-row-last-activity">
-          {server.last_activity ? timeSince(server.last_activity) : "Never"}
+          {user.last_activity ? timeSince(server.last_activity) : "Never"}
```

Comment:
Change both the string and the condition:

```suggestion
          {user.last_activity ? timeSince(user.last_activity) : "Never"}
```

But this is going to use the user activity for _all_ servers, which I think isn't what we want.

Maybe what we want is really:

1. user.lastActivity _if_ server.name === "" _and_ server.last_activity is null.
2. else server.lastActivity

Which is probably best encapsulated in a new variable, around 226

```javascript
// show server activity for running servers (always for named servers)
// and user activity in the default row if the default server isn't running
const lastActivity = (server.lastActivity || server.name !== "") ? server.lastActivity : user.lastActivity;
```

What do you think?

Reply:
Sounds fine by me. 

In my setup, no user ever uses other server than its own, so I don't think there is a use case where this would make a difference (also sorry for the wrong PR obviously).